pkg/cmd/backuprestore/restoreutils.go

```diff
@@ -14,6 +15,17 @@ var static_pod_list = []string{
 	"kube-scheduler-pod.yaml",
 }
 
+var exists = struct{}{}
+
+var staticPodContainers = map[string]struct{}{
+	"etcd":                    exists,
+	"etcdctl":                 exists,
+	"etcd-metrics":            exists,
+	"kube-controller-manager": exists,
+	"kube-apiserver":          exists,
+	"kube-scheduler":          exists,
+}
+
 func restore(configDir, dataDir, backupDir string) error {
```

Comment:
This probably needs to receive a context to support cancellation (at the least to account for signal handlers)

Reply:
Done.